underfs/s3a/src/test/java/alluxio/underfs/s3a/S3AUnderFileSystemTest.java

```diff
@@ -239,4 +239,11 @@ public void getOperationMode() throws Exception {
     Assert.assertEquals(UfsMode.READ_WRITE,
         mS3UnderFileSystem.getOperationMode(physicalUfsState));
   }
+
+  @Test
+  public void stripPrefixIfPresent() throws Exception {
+    Assert.assertEquals("", mS3UnderFileSystem.stripPrefixIfPresent("s3a://" + BUCKET_NAME));
+    Assert.assertEquals("", mS3UnderFileSystem.stripPrefixIfPresent("s3a://" + BUCKET_NAME + "/"));
+    Assert.assertEquals("test/", mS3UnderFileSystem.stripPrefixIfPresent("test"));
```

Comment:
Should there be 1 additional case of something like "s3a://" + BUCKET_NAME + "/path/" ?

Reply:
ah right yeah I'll do that.